pytest_fixtures/component/provision_pxe.py

```diff
@@ -205,18 +204,24 @@ def module_ssh_key_file():
 def provisioning_host(module_ssh_key_file):
     """Fixture to check out blank VM"""
     vlan_id = settings.provisioning.vlan_id
-    vm_firmware = "bios"  # TODO: Make this a fixture parameter - bios, uefi
+    vm_firmware = "uefi"  # TODO: Make this a fixture parameter - bios, uefi
     cd_iso = ""  # TODO: Make this an optional fixture parameter
-    with Broker(
-        workflow="deploy-configure-pxe-provisioning-host-rhv",
-        host_class=ContentHost,
-        target_vlan_id=vlan_id,
-        target_vm_firmware=vm_firmware,
-        target_vm_cd_iso=cd_iso,
-        blank=True,
-        target_memory='6GiB',
-        auth=module_ssh_key_file,
-    ) as host:
-        yield host
-        # Set host as non-blank to run teardown of the host
-        host.blank = False
+    try:
+        with Broker(
+            workflow="deploy-configure-pxe-provisioning-host-rhv",
+            host_classes={'host': ContentHost},
+            target_vlan_id=vlan_id,
+            target_vm_firmware=vm_firmware,
+            target_vm_cd_iso=cd_iso,
+            blank=True,
+            target_memory='6GiB',
+            auth=module_ssh_key_file,
+        ) as prov_host:
+            yield prov_host
+            # Set host as non-blank to run teardown of the host
+            prov_host.blank = False
+    except HostError as error:
+        if "Temporary failure in name resolution" in error.message:
```

Comment:
Checking in a host before provisioning has finished. In this test I end it when I see the host calling for the kickstart repo, since the rest of the coverage should be handled by other provisioning tests.

Reply:
A little more detail: So when you try to checkin the host, it will actually be removed in RHV, but you will get the error above. So we just pass the error.

Reply:
Gotcha, the test logic and steps make sense to me. So if this error comes from AAP, can you catch the error there instead of here in robottelo?

Reply:
The error comes from subscription-manager unregister. It tries to reach out to the host, but it's unreachable. We attempted to use `ignore_errors` there, but that didn't change anything. The WF will continue to execute after that, but broker will still send you the "ignored" error.